Saritasa.Cake.sh

```diff
@@ -0,0 +1 @@
+dotnet run --project ./scripts/Modules/Saritasa.Cake/Saritasa.Cake.csproj -- "$@"
```

Comment:
Please update readme.md / install.md documents on how to use the new scripts.
Also, if everything is working fine, we can remove the other powershell scripts from this repo?